api/api_public_transaction_pool.go

```diff
@@ -323,6 +336,26 @@ func (s *PublicTransactionPoolAPI) SendTransaction(ctx context.Context, args Sen
 	return submitTransaction(ctx, s.b, signedTx.Tx)
 }
 
+// SendTransactionAsFeePayer creates a transaction for the given argument, sign it as a fee payer
+// and submit it to the transaction pool.
+func (s *PublicTransactionPoolAPI) SendTransactionAsFeePayer(ctx context.Context, args SendTxArgs) (common.Hash, error) {
+	// Don't allow dynamic assign of the nonce value since the sender already signed on a specific nonce value.
+	if args.Nonce == nil {
```

Comment:
Don't we need to check gasPrice and gasLimit also?

Reply:
Thank you. I changed it to return errors for the values set by `setDefaults`